### PR TITLE
append: Documented that it accepts text nodes

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
Fixes gh-879

### CODE REVIEW

File: pages/Types.html [hunk 1]
```diff
@@ -604,6 +605,17 @@ <h2 id="Element"> Element </h2>
...
+<h2 id="Text"> Text </h2>
+<p>Text is a node of the Document Object Model (DOM) that represents the textual content of an <a href="#Element">element</a> or an attribute. Consider the following code:
+</p>
+<pre><code data-lang="html">&lt;p id=&quot;target&quot;&gt;&lt;b&gt;Hello&lt;/b&gt; world&lt;/p&gt;
+</code></pre>
+<p>If you retrieve the children of the paragraph of the example as follows:
+</p>
+<pre><code data-lang="javascript">var children = document.getElementById( "target" ).childNodes;
+</code></pre>
+you obtain two children. The first one is the <a href="#Element">element</a> represeting the <code>b</code> tag. The second children is a text node containing the string " world".
```
Comment (agcolom) [inline pages/Types.html]: typo: represeting -> representing
also: the second **children** should be the second **child**

File: pages/Types.html [hunk 2]
```diff
@@ -604,6 +605,17 @@ <h2 id="Element"> Element </h2>
 </code></pre>
 <p>You could replace <code>this.value</code> with <code>$(this).val()</code> to access the value of the text input via jQuery, but in that case you wouldn't gain anything.
 </p>
+<h2 id="Text"> Text </h2>
+<p>Text is a node of the Document Object Model (DOM) that represents the textual content of an <a href="#Element">element</a> or an attribute. Consider the following code:
```
Comment (agcolom) [inline pages/Types.html]: @kswedberg Could you please have a look. I'm wondering whether there should be a comma before 'or'. Thanks.

Comment (kswedberg) [inline pages/Types.html]: @agcolom It's fine as is. No comma needed.

**Comment by AurelioDeRosa** [comment]
Cool. So is it good to merge?

**Comment by agcolom** [comment]
LGTM :-)

**Pull request closed without merging**